components/scream/src/physics/shoc/shoc_diag_second_moments_ubycond.cpp

```diff
@@ -0,0 +1,16 @@
+#include "shoc_diag_second_moments_ubycond_impl.hpp"
+#include "ekat/scream_types.hpp"
+
+namespace scream {
+namespace shoc {
+
+/*
+ *  * Explicit instantiation for doing ice melting on Reals using the
```

Comment:
Can you get rid of the "ice melting" comment here? Let's keep this generic.

Reply:
I didn't realize this one, modification is done! please check it.